workspaces/linux-host/src/linux-host/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+authors = ["The Veracruz Development Team"]
+description = "A dummy package"
+edition = "2018"
+name = "linux-host"
+version = "0.3.0"
+
+[dependencies]
+tests = { path = "../../crates/tests", features = [ "linux" ] }
```

Comment:
This dependency on `tests` gives a warning, I think, though removing it leads to an error. Not sure what's best!

```
warning: linux-host v0.3.0 (/work/veracruz/workspaces/linux-host/src/linux-host) ignoring invalid dependency `tests` which is missing a lib target
```

Reply:
I forgot to commit a file (empty `tests/src/lib.rs`). That should be fixed now.